packages/@okta/vuepress-site/.vuepress/nav/guides.js

```diff
@@ -5,6 +5,7 @@ module.exports = [
   { 
     title: 'Basics', 
     subLinks: [
+      { title: 'QuickStart', guideName: 'quickstart' },
```

Comment:
Let's change the title to _Quickstart: Signing in your first user_, here and anywhere else the title appears.